src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,37 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name, health, hidden = false) {
+    super(name, health);
+    this.hidden = hidden;
```

Comment:
[CODE PERFORMANCE] The constructor of Herbivore should not accept a `health` parameter since it is always initialized to `100` by default in the Animal class. Remove the `health` parameter to prevent unnecessary complexity and potential bugs.